app/components/UI/TooltipButton.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box, Text } from 'rebass'
+
+const StyledTooltipIconBox = styled(Box)`
+  border-radius: 50%;
+  height: 16px;
+  line-height: 16px;
+  margin-left: 5px;
+  position: relative;
+  text-align: center;
+  transform: translateZ(0);
+  width: 16px;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+  }
+`
+StyledTooltipIconBox.displayName = 'TooltipIconBox'
+
+const StyledTooltipWrapper = styled(Box)`
+  border: 1px solid white;
+  border-radius: 4px;
+  padding: 5px 5px 5px 25px;
+  position: absolute;
+  top: -5px;
+  width: 225px;
+  z-index: 1;
+  transform: translateZ(0);
+`
+StyledTooltipWrapper.displayName = 'TooltipWrapper'
+
+/**
+ * @render react
+ * @name TooltipButton
+ * @example
+ * <TooltipButton>Some explanation text here</TooltipButton>
+ */
+class TooltipButton extends React.Component {
+  static displayName = 'TooltipButton'
+
+  state = { hover: false }
+
+  hoverOn = () => {
+    this.setState({ hover: true })
+  }
+
+  hoverOff = () => {
+    this.setState({ hover: false })
+  }
+
+  render() {
+    const { children, ...rest } = this.props
+    const { hover } = this.state
+
+    return (
+      <Box
+        style={{
+          position: 'relative'
+        }}
+      >
+        <StyledTooltipIconBox
+          bg="white"
```

Comment:
Should be `primaryText` to keep it compatible with other themes

Reply:
Will do.